packages/ui/src/components/Forms/DateTimeInput/FallbackDateTimeInput.tsx

```diff
@@ -0,0 +1,82 @@
+import cn from 'classnames'
+import { ChangeEvent, forwardRef, memo, Ref, useCallback, useEffect, useMemo, useRef, useState } from 'react'
+import { VisuallyHidden } from 'react-aria'
+import { useComponentClassName } from '../../../auxiliary'
+import { toEnumStateClass, toEnumViewClass, toViewClass } from '../../../utils'
+import { Stack } from '../../Stack'
+import { TextInputOwnProps } from '../TextInput'
+import { dateToDateValue, dateToTimeValue } from './Serializer'
+
+function setNativeValue(element: HTMLInputElement, value: string) {
+	const valueSetter = Object.getOwnPropertyDescriptor(element, 'value')?.set
+	const prototype = Object.getPrototypeOf(element)
+	const prototypeValueSetter = Object.getOwnPropertyDescriptor(prototype, 'value')?.set
+
+	if (valueSetter && valueSetter !== prototypeValueSetter) {
+		prototypeValueSetter?.call(element, value)
+	} else {
+		valueSetter?.call(element, value)
+	}
+
+	element.dispatchEvent(new Event('input', { bubbles: true }))
+}
```

Comment:
Why???

Reply:
React won't emit change event when value is being changed via prop.